Fix yamllint warnings, enable strict mode

*This is a good first issue for new contributors to take on, if you have any questions, please ask on the task or in our [Gitter room](https://gitter.im/freedomofpress/securedrop)!*

## Description

We currently have some yamllint warnings:

```
$ make yamllint
███ Linting YAML files...
./.circleci/config.yml
  78:221    warning  line too long (293 > 220 characters)  (line-length)
  256:221   warning  line too long (222 > 220 characters)  (line-length)
  262:221   warning  line too long (224 > 220 characters)  (line-length)

./install_files/ansible-base/roles/validate/tasks/validate_tails_environment.yml
  81:4      warning  wrong indentation: expected 4 but found 3  (indentation)

./install_files/ansible-base/roles/restore/tasks/perform_restore.yml
  112:9     warning  too many spaces after hyphen  (hyphens)

./install_files/ansible-base/group_vars/securedrop_application_server.yml
  12:5      warning  too many spaces after hyphen  (hyphens)
```

These should be fixed. Then we can enable `--strict` mode (in `devops/scripts/yaml-lint.sh`) so that any future warnings will cause CI to fail.

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -75,7 +75,8 @@ jobs:
           name: Run all linters
           command: |
             fromtag=$(docker images |grep securedrop-test-focal-py3 |head -n1 |awk '{print $2}')
-            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-focal-py3:${fromtag:-latest}" securedrop/bin/dev-shell bash -c "sudo apt install shellcheck && /opt/venvs/securedrop-app-code/bin/pip3 install --require-hashes -r requirements/python3/develop-requirements.txt && make -C .. lint"
+            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-focal-py3:${fromtag:-latest}" securedrop/bin/dev-shell \
+                  bash -c "sudo apt install shellcheck && /opt/venvs/securedrop-app-code/bin/pip3 install --require-hashes -r requirements/python3/develop-requirements.txt && make -C .. lint"
 
   app-tests:
     machine:
@@ -247,19 +248,22 @@ jobs:
           name: Check Python dependencies for CVEs
           command: |
             fromtag=$(docker images |grep securedrop-test-focal-py3 |head -n1 |awk '{print $2}')
-            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-focal-py3:${fromtag:-latest}" securedrop/bin/dev-shell bash -c "pip3 install -U -q --upgrade safety && make -C .. safety"
+            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-focal-py3:${fromtag:-latest}" securedrop/bin/dev-shell \
+                  bash -c "pip3 install -U -q --upgrade safety && make -C .. safety"
 
       - run:
           name: Run static security testing on source code with bandit
           command: |
             fromtag=$(docker images |grep securedrop-test-focal-py3 |head -n1 |awk '{print $2}')
-            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-focal-py3:${fromtag:-latest}" securedrop/bin/dev-shell bash -c "pip3 install -U -q --upgrade pip && pip3 install -U -q --upgrade bandit && make -C .. bandit"
+            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-focal-py3:${fromtag:-latest}" securedrop/bin/dev-shell \
+                  bash -c "pip3 install -U -q --upgrade pip && pip3 install -U -q --upgrade bandit && make -C .. bandit"
 
       - run:
           name: Run static security testing on source code with semgrep
           command: |
             fromtag=$(docker images |grep securedrop-test-focal-py3 |head -n1 |awk '{print $2}')
-            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-focal-py3:${fromtag:-latest}" securedrop/bin/dev-shell bash -c "pip3 install -U -q --upgrade pip && pip3 install -U -q --upgrade semgrep && make -C .. semgrep"
+            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-focal-py3:${fromtag:-latest}" securedrop/bin/dev-shell \
+                  bash -c "pip3 install -U -q --upgrade pip && pip3 install -U -q --upgrade semgrep && make -C .. semgrep"
 
   staging-test-with-rebase:
     machine:

--- a/devops/scripts/yaml-lint.sh
+++ b/devops/scripts/yaml-lint.sh
@@ -12,4 +12,4 @@ find . -path "*/.venv*" -prune -o \
                  -path "**/.molecule" -prune -o \
                  -type f | \
         grep -E '^.*\.ya?ml' | \
-        xargs yamllint -c ".yamllint"
+        xargs yamllint --strict -c ".yamllint"

--- a/install_files/ansible-base/group_vars/securedrop_application_server.yml
+++ b/install_files/ansible-base/group_vars/securedrop_application_server.yml
@@ -9,7 +9,7 @@ local_deb_packages:
   - "securedrop-keyring-0.1.6+{{ securedrop_version }}+{{ securedrop_target_distribution }}-amd64.deb"
   - "securedrop-config-0.1.4+{{ securedrop_version }}+{{ securedrop_target_distribution }}-amd64.deb"
   - "securedrop-ossec-agent-3.6.0+{{ securedrop_version }}+{{ securedrop_target_distribution }}-amd64.deb"
-  -  securedrop-grsec-{{ grsec_version }}+{{ securedrop_target_distribution }}-amd64.deb
+  - "securedrop-grsec-{{ grsec_version }}+{{ securedrop_target_distribution }}-amd64.deb"
   - "{{ securedrop_app_code_deb }}.deb"
   - "ossec-agent-3.6.0+{{ securedrop_target_distribution }}-amd64.deb"
 

--- a/install_files/ansible-base/roles/restore/tasks/perform_restore.yml
+++ b/install_files/ansible-base/roles/restore/tasks/perform_restore.yml
@@ -109,7 +109,7 @@
     exclude:
       - "var/lib/tor/services/source,var/lib/tor/services/journalist"
       - "var/lib/tor/services/ssh"
-      -  "var/lib/tor/services/sshv3"
+      - "var/lib/tor/services/sshv3"
   when: (not restore_skip_tor) and
         ("V3 services only" in compare_result.stdout)
 

--- a/install_files/ansible-base/roles/validate/tasks/validate_tails_environment.yml
+++ b/install_files/ansible-base/roles/validate/tasks/validate_tails_environment.yml
@@ -78,7 +78,7 @@
       file under `~/Persistent/securedrop/install_files/ansible-base/ and
       retry the install command.
   when:
-   - v3_source_file.stat.exists
+    - v3_source_file.stat.exists
 
 - name: Confirm that the Tor keys file is present
   assert:

